doc/api/fs.md

```diff
@@ -1583,6 +1583,29 @@ fs.open('<directory>', 'a+', (err, fd) => {
 });
 ```
 
+Some characters are reserved under Windows which documented by
+[Naming Files, Paths, and Namespaces][].
+
++ `<` (less than)
```

Comment:
I'm honestly not a huge fan of documenting the operating system like this in Node's docs. Since we're already linking to MSDN, this list can probably be dropped.